packages/herm-js-document/modules/index.js

```diff
@@ -88,16 +88,19 @@ class HermDoc {
     }
 
     result.content = Object.assign([], this.content, newDoc.content);
-    
+
     return result;
   }
 
-  deleteRange(from, to) {
-  	let r = to - from;
-    for(let i = 0 ; i < r; i++) {
-    	const k = this.keys[from + i];
-      this.content[k] =  null;
+  deleteRange(from, length) {
+    for(let i = 0; i < length; i++) {
+      const k = this.keys[from + i];
+
+      delete this.content[k];
```

Comment:
I was leaving the value to null because I need to change the merge function (L90) to test for those null values and not add back something you already removed